loops.js

```diff
@@ -6,6 +6,14 @@
 // Hoera!
 // ==========================================
 
+for (let i = 0; i < 99; i++) {
+    console.log("(●'◡'●)");
+}
```

Comment:
Hahahaha, leuk !!! Is dit een opdracht die ik gemist heb?!
100x een glimlach vind ik altijd de moeite Jelmer !!!